### PR TITLE
Reservations

### PR DESCRIPTION
**Pull request by serenaqchen**
## Overview

Created a reservations page where customers are able to view weather before making a reservation for outdoor dining. (currently still working on accessibility & making it more mobile-friendly).

## Feedback Requested

- structure of code 
- logic of fxns 
- accessibility

### CODE REVIEW

**Comment by ZelmaSedano** [comment]
Good job fixing heroku! Looks good :)

File: app/src/Reservations/Reservations.js [hunk 1]
```diff
@@ -1,9 +1,66 @@
...
+  const [fiveDayForecast, setFiveDayForecast] = React.useState(false);
+
+  const handleForecastButton = (e) => {
+    return fiveDayForecast
+      ? setFiveDayForecast(true)
+      : setFiveDayForecast(false);
+  };
+
+  const loadCurrentWeather = async () =>
+    setCurrentWeather(await apiClient.getCurrentWeather());
```
Comment (malikasen) [inline app/src/Reservations/Reservations.js:23]: I got a recommendation from my mentor, to create different API clients for each route. You can do that in the future. For example, one for weather, another one for your menu data from the back-end and so on

File: app/src/Reservations/weatherFxns.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+};
+
+export const dateConversion = (date) => {
+  const options = {
+    year: "numeric",
+    month: "numeric",
+    day: "numeric",
+  };
+  let dt = new Date(date * 1000);
+  return dt.toLocaleDateString("en-US", options);
```
Comment (malikasen) [inline app/src/Reservations/weatherFxns.js:24]: good job manipulating date data!

File: app/src/Reservations/weatherFxns.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+  if (atmosphereConditions.includes(weatherCondition)) {
+    return (
+      <img
+        className="conditionIcon"
+        src={atmosphere}
+        alt={weatherCondition}
+      ></img>
+    );
+  } else if (weatherCondition === "Clear") {
```
Comment (malikasen) [inline app/src/Reservations/weatherFxns.js:49]: Can't you fetch icons from the API directly?

**Review by malikasen — APPROVED** [review]
Good progress!

**Pull request merged**